tools/offsets.py

```diff
@@ -122,6 +122,45 @@ def __contains__(self, address: int) -> bool:
         "overlay34": Binary(0x22DCB80, 0xDC0),
         "overlay35": Binary(0x22BD3C0, 0x20),
     },
+    "JP": {
+        "arm9": Binary(0x2000000, 0x756920),
```

Comment:
This length is wrong, should be 0xB8CB8 based on the YAML file updates.

Reply:
Yep, no idea how I got this value.